### DISCUSSION TITLE
CairoSVG

### DISCUSSION THREAD

**Discussion by oreactko** [BirdLogics/sb3topy#15] [Q&A]
I have this error: 
![image](https://github.com/user-attachments/assets/11abdfd7-0b92-43b8-85d6-c268ccad2e79)
but:
![image](https://github.com/user-attachments/assets/fce0ced0-16d7-4bda-8e91-58cbfd943724)

**Comment by BirdLogics** [comment]
Is it possible that you are running sb3topy with a different Python version than what pip is using? It might help if you use pip3 rather than just pip, depending on your installation.

With the latest release of sb3topy, SVG conversion is optional. You can also just use Pygame's built-in SVG support. The resolution will be a worse though. You should be able to disable SVG conversion in settings to hide the error.
